packages/Liquid-Core.package/LiquidParticipantMenu.class/instance/buildWith..st

```diff
@@ -1,14 +1,13 @@
 initialize
 buildWith: aBuilder 
 
+self buildAnswerPanelWith: aBuilder.
+
 ^ aBuilder build: (aBuilder pluggableWindowSpec new
 	model: self;
 	label: 'Liquid - Participant';
 	extent: 500@400;
-	children: { self buildQuestionListWith: aBuilder.
-					self buildChoiceListWith: aBuilder.
-					self buildVotedChoiceListWith: aBuilder.
-					self buildVoteChoiceButtonWith: aBuilder.
+	children: {	self answerPanel.
```

Comment:
Is an instance variable neccessary here? Could we have `buildAnswerPanelWith:` return this instead?

Reply:
If we drop the instance variable, we have no way of reaching the `AnswerPanel` from the `ParticipantMenu`, which would mean we cannot modify any questions (or answers) after a poll was chosen. At this point we do not need to (to my knowledge), but given that multiple questions in one poll as well as modifications to running polls are features that are very likely to be implemented later on, I would consider leaving it, as both of those need access to the `AnswerPanel`.

Reply:
YAGNI. Whoever needs that instance variable will add it once they need it.